WMF Framework/UIScrollView+Limits.swift

```diff
@@ -18,11 +18,13 @@ extension UIScrollView {
     }
 
     public var isAtTop: Bool {
-        return contentOffset.y <= topOffsetY
+        /// 0.5 is to account for  rounding errors. Sometimes when we expect them to be equal, these are less than .2 different (due to rounding) - and with multiple layout passes, it caused a large scrolling bug on a VC's launch.
+        return contentOffset.y <= topOffsetY + 0.5
```

Comment:
What a strange bug! Is there special significance to using `0.5`? If not, what do you think about using `topOffsetY.rounded(.up)` instead, so we're at the next integral number without fear of more rounding errors?

Reply:
No significance to `0.5` - just big enough to seem safe. You're absolutely correct, rounding is much cleaner. Updated this.